common/pom.xml

```diff
@@ -54,5 +53,13 @@
             <groupId>org.mockito</groupId>
             <artifactId>mockito-core</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-simple</artifactId>
+        </dependency>
```

Comment:
Do you need this dependency for testing? It looks like the slf4j-api is already available.